servlet/src/main/java/io/undertow/servlet/UndertowServletMessages.java

```diff
@@ -244,4 +244,10 @@ public interface UndertowServletMessages {
 
     @Message(id = 10066, value = "Can not set invoke 'declareRoles' from dynamic listener in servlet context for context path '%s' in deployment '%s' ")
     UnsupportedOperationException cantCallFromDynamicListener(String deploymentName, String contextPath);
+
+    @Message(id = 10067, value = "Servlet container does not permit session creation after reponse was committed.")
```

Comment:
I just noticed this reponse instead of response typo. I'm updating the PR so we can finally merge it!